script.js

```diff
@@ -0,0 +1,18 @@
+module.exports = core => {
+  const { RUNNER_OS, UPDATE_TIMES_A_DAY } = process.env;
+  const times = Number(UPDATE_TIMES_A_DAY);
+
+  // calculate the docker-key cache value
+  if (UPDATE_TIMES_A_DAY) {
```

Comment:
it looks better to keep if's as small as we can.. so please instead of put all logic inside the if -> 
if(!UPDATE_TIMES_A_DAY) => finish
// logic goes here

Reply:
done